### ISSUE TITLE
GREP_OPTIONS is deprecated

### ISSUE THREAD

**Issue by mangroovie** [miohtama/ztanesh#31]
Gets grep: warning: GREP_OPTIONS is deprecated
when using grep
...Issue since grep 2.20 (at least on Ubuntu 15.11 and 16.04)

**Comment by ztane** [comment]
Yes, and it also breaks some stuff like `pip install matplotlib`
